protoio/uvarint_test.go

```diff
@@ -148,3 +141,189 @@ func iotest(writer protoio.WriteCloser, reader protoio.ReadCloser) error {
 	}
 	return nil
 }
+
+// Everything below this line is borrowed from the generated test code in gogoproto
```

Comment:
`gogoproto` has some magic options to generate test helpers. Now that we are not using `gogoproto` I had to recreate their test fixtures, generate the code with Google Protobuf, and then copy and paste over the test helpers below. It's not pretty.